plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -39,8 +39,9 @@ function DataTable(element) {
 
 DataTable._footerIconHandlers = {};
 
-DataTable.initNewDataTables = function () {
-    $('div.dataTable').each(function () {
+DataTable.initNewDataTables = function (reportId) {
+    var selector = reportId ? '[data-report="'+reportId+'"]' : 'div.dataTable';
```

Comment:
not sure if we need to escape the selector? Not sure if we have any user generated reportIds or so that could somehow cause issues?